test/shape/list.test.ts

```diff
@@ -45,12 +45,14 @@ describe('org:shape:list', () => {
 
   it('no shapes', async () => {
     await $$.stubAuths(testOrg);
-    sandbox.stub(OrgShapeListCommandFunctions, 'getAllOrgShapesFromAuthenticatedOrgs').resolves([]);
+    sandbox
+      .stub(OrgShapeListCommandFunctions, 'getAllOrgShapesFromAuthenticatedOrgs')
+      .resolves({ orgShapes: [], errors: [] });
 
     const command = new OrgShapeListCommand([], config);
     await command.run();
-    expect(uxLogStub.calledOnce).to.be.true;
-    expect(uxLogStub.firstCall.args[0]).to.include('No org shapes found.');
+    expect(uxLogStub.calledTwice).to.be.true;
+    expect(uxLogStub.secondCall.args[0]).to.include('No org shapes found.');
```

Comment:
these have been bugging me because we keep having to change them.

Can the assert be (I forget the exact syntax)  uxLogStub.getCalls().args.some( a => a.includes(...)) so that it survives various ux tweaks?